### ISSUE TITLE
Clear/refresh page/element cache while using REDIS on NGINX does not work as expected

### ISSUE THREAD

**Issue by TomHell** [putyourlightson/craft-blitz#143]
Hi Ben,
I have change my setup from local caching to REDIS.

**I have done:**
- changed app.php and define credentials for REDIS, SESSION, CACHE and QUEUE
- changed the Cache-Storage in Blitz-Plugin to yii2/redis/cache

**Problem**
I`m saving new content for my page. The backend save the new content but in the frontend is still the old cached version available. I can see activities in REDIS database by saving the entry. Also **cache/clear** works in console or browser very well and than I see the new version of my page in the frontend. 
- no error in the Chrome Console while saving the entry
- logs does not have any errors

When I switch back to Local-Caching everything works great and there is no issue.

**Software**:
- CraftCMS 3.1.34.3
- Blitz Plugin 2.0.10
- Queue Manager 1.1.0
- much more 

**Setup**
- NGINX 1.17
- PHP 7.2, also tested with PHP 7.3
- REDIS
- MySQL Database 5.7
- removed configuration for nginx config file (tip from you)
- also remove web/cache folder (tip from you)

**Comment by bencroker** [comment]
Hi Tom and thanks for the detailed issue description. I'll run some tests here to see if I can replicate locally. Is there a reason you need to still be running Craft 3.1.x and Blitz 2.0.x and cannot update them?

**Comment by TomHell** [comment]
Hi Ben,

I have updated Craft CMS and your Blitz Plugin, also all other dependencies of my project. After all everything works as expected. Thanks for your tip.

- Craft CMS 3.3.15
- Blitz Plugin, 2.3.5

The only thing that I noticed is that after all updating my Queue Manager was filled up with 2000 and more jobs and I could not save my entries (content).
Known Issue => https://github.com/craftcms/cms/issues/4507

**Possible Solution**
I canceled all jobs in the Queue Manager and had to restart my environment (NGINX + PHP).
I cleared all caches, (Craft CMS, Blitz Plugin). After that I could save my entries (content) again and the system works well again.
